src/api/crosscutRequests.js

```diff
@@ -39,27 +45,44 @@ export const fetchCatchmentJobs = async () => {
     }
 }
 
-export const createCatchmentJob = async (json) => {
-    const url = `${CROSSCUT_API}/catchment-jobs`
-    try {
-        const id = json.level
-        const geojson = await fetchGeoJSON(id)
-    
-        // TO-DO: needs name, lat, lng, csv
-        // const resp = await ky.post(url, {
-        //     json,
-        //     mode: "cors",
-        //     headers: {
-        //       authorization: getToken(),
-        //     },
-        // })       
-    } catch (err) {
-        throw err
+export const createCatchmentJob = async (body) => {
+    const url = `${baseURL}/catchment-jobs`
+    const levelId = body.level
+    const groupId = body.group
+
+    let data = null
+    let csv = body.csv
+    if (csv === "") {
+        data = await fetchValidPoints(levelId, groupId)
+        csv = papaparse.unparse(data)
+    } else {
+        csv = papaparse.unparse(body.csv)
     }
+
+    const json = {
+        fields: {
+            lat: "lat",
+            lng: "long",
+            name: "name"
+        },
+        name: body.name,
+        country: body.country,
+        csv,
+        algorithm: body.algorithm
+    }
+
+    // TO-DO: there needs to be a way to save the fields they chose (levels and groups)
```

Comment:
I think it would be good to be consistent -- some places say `TO-DO` but others say `TODO` -- I've usually seen TODO comments as `TODO`